app-office/hledger-iadd/hledger-iadd-1.3.21-r2.ebuild

```diff
@@ -17,7 +17,9 @@ LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
 
-RDEPEND=">=dev-haskell/brick-2.1:=[profile?] <dev-haskell/brick-2.7:=[profile?]
+CABAL_CHDEPS=('brick >= 2.1 && < 2.7' 'brick >= 2.1 && < 2.9')
+
+RDEPEND=">=dev-haskell/brick-2.1:=[profile?] <dev-haskell/brick-2.9:=[profile?]
```

Comment:
Thank you. Note there is an upstream hackage revision that also achieves this, but that's fine - this is equivalent.